tornjak-frontend/src/auth/KeycloakAuth.js

```diff
@@ -0,0 +1,54 @@
+import Keycloak from "keycloak-js";
+const keycloakConfig = {
+    "realm": "tornjak",
+    "url": process.env.REACT_APP_AUTH_SERVER_URI,
+    "ssl-required": "external",
+    "clientId": "Tornjak-React-auth",
+    "public-client": true,
+    "verify-token-audience": true,
+    "use-resource-role-mappings": true,
+    "confidential-port": 0
+};
```

Comment:
does this mean the library handles the storage of the client-secret for Authorization code flow?

Reply:
yes